src/DynamoCore/Configuration/IPreferences.cs

```diff
@@ -124,4 +138,29 @@ public interface IPreferences
         /// otherwise.</returns>
         bool Save(string filePath);
     }
+
+    /// <summary>
+    /// Represents data about active state of preview background
+    /// </summary>
+    public class BackgroundPreviewActiveState
```

Comment:
deserialization of List<BackgroundPreviewActiveState> works fine without additional attributes